notifiers/telegramNotifier_test.go

```diff
@@ -0,0 +1,338 @@
+package notifiers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-keys-monitor-go/core"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/stretchr/testify/assert"
+)
+
+const testTelegramToken = "test-token"
+const testTelegramChatID = "test-chat-id"
+
+func createHttpTestServerThatRespondsOKForTelegram(
+	t *testing.T,
+	expectedMessage string,
+	expectedTitle string,
+	numCalls *uint32,
+) *httptest.Server {
+	response := &pushoverResponse{
```

Comment:
This response object is specific to pushover. Maybe use one for TG?

Reply:
copy-paste error. Removed